commands/alias.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractCommand, GlobalState } from './abstractCommand'
+import { checkPeerIdInput } from '../utils'
+
+const COMMAND_FORMAT = /([A-Za-z0-9]{53})\s([A-Za-z0-9]*)/
```

Comment:
Don't we have something in utils that helps us check that a string is a HOPR Address? Otherwise, this is fine, but worth storing somewhere.

Reply:
yes, we subsequently validate the address with checkPeerID - this just makes sure it's the right length.